corehq/apps/toggle_ui/views.py

```diff
@@ -276,6 +290,9 @@ def _set_toggle(request_username, static_toggle, item, namespace, is_enabled):
         _clear_cache_for_toggle(namespace, item)
         _enable_dependencies(request_username, static_toggle, item, namespace, is_enabled)
 
+    if static_toggle == ATTENDANCE_TRACKING:
```

Comment:
@SmittieC one more thing: We should be using `StaticToggle.save_fn` to do this. See `CASE_LIST_EXPLORER`, `CASE_API_V0_6`, and `COMMTRACK` for examples.